pkg/extensions/search/resolver.go

```diff
@@ -1166,7 +1171,10 @@ func expandedRepoInfo(ctx context.Context, repo string, repoDB repodb.RepoDB, cv
 					break
 				}
 
-				manifestMetaMap[descriptor.Digest.String()] = manifestMeta
+				manifestMetaMap[descriptor.Digest.String()] = repodb.ManifestMetadata{
```

Comment:
ManifestMetadata is being used as a way to retrieve ManifestBlob and ConfigBlob all thorough the project. In this case the ManifestMeta will be used by the convert functions that don't use any other field other than the ones I mentioned. Calling `repoDB.GetManifestMeta()` simply does more DB queries.

Besides the repoDB tests we can simply search and replace from ManifestMetadata to ManifestData and it would be equivalent.